tools/dhis2-audit-data-extractor/README.md

```diff
@@ -11,14 +23,61 @@ $ pip install -r requirements.txt
 N.B. You may need to install the package `libpq-dev`. Please refer to your distro's documentation on how to install it.
 
 ## Run
-As simple as:
 ```
-$ python extract_audit.py
+~/dhis2-utils/tools/dhis2-audit-data-extractor$ python extract_audit.py
+usage: extract_audit.py [-h] [-e ENTRIES] [-m {file,stdout}] [-f {CSV,JSON}] [-s SKIP] [-o OUTPUT] [{extract,enum}]
+
+positional arguments:
+  {extract,enum}
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -e ENTRIES, --entries ENTRIES
+                        Number of rows to pull. Default 1000
+  -m {file,stdout}, --mode {file,stdout}
+  -f {CSV,JSON}, --format {CSV,JSON}
+  -s SKIP, --skip SKIP  Number of rows to skip
+  -o OUTPUT, --output OUTPUT
+                        Output file
+~/dhis2-utils/tools/dhis2-audit-data-extractor$
+```
+
+You can enumerate how many events (=rows) there are in the table:
+```
+~/dhis2-utils/tools/dhis2-audit-data-extractor$ python extract_audit.py enum
+Audit table contains 612 entries
+~/dhis2-utils/tools/dhis2-audit-data-extractor$
+```
+
+By default the script saves the data to a file (`-m file`) automatically created in the current folder. Format is CSV (`-f CVS`).
+
+```
+$ python extract_audit.py extract
+```
+
+With large tables, you may want to extract only a subset of events. You can do it by settin the `--entries` and `--skip` paramenters. The command below pulls 100 events starting from row 10:
+```
+$ python extract_audit.py extract -e 100 -s 10
+```
+
+You can append the data to an already existing file the following way:
+```
+$ python extract_audit.py extract -o results.csv -e 10
+$ python extract_audit.py extract -o results.csv -e 10 -s 10
 ```
 
-You may want to chain the output with `jq` for better results:
+Bear in mind that while selecting the JSON mode, a JSON object will be appended as a standalone entity, thus not merged into one single JSON.
 ```
-$ python extract_audit.py | jq -r .
+$ python extract_audit.py extract -f JSON -o results.json -e 10
+Data saved in results.json
+$ cat results.json | jq length
+10
+$ python extract_audit.py extract -f JSON -o results.json -e 10 -s 10
+Data saved in results.json
+$ cat results.json | jq length
+10
+10
+$
```

Comment:
The documentation and examples you've written are super clear and easy to understand :+1:  :star2: